gcloud-java-examples/src/main/java/com/google/cloud/examples/storage/StorageExample.java

```diff
@@ -512,6 +563,188 @@ public String params() {
     }
   }
 
+  private abstract static class AclAction extends StorageAction<Tuple<BlobId, Acl>> {
+
+    @Override
+    public void run(Storage storage, Tuple<BlobId, Acl> params) {
```

Comment:
This should have some explanation that we have two paths, one for just the bucket, and one for an actual blob.  We are showing both in this example.